src/lib-imap/imap-capability-list.c

```diff
@@ -0,0 +1,129 @@
+#include "imap-capability-list.h"
+
+/* initialize the imap capability list */
+struct imap_capability_list *
+imap_capability_list_create(const char *initial_entries)
+{
+    /* initialize a capability list object */
```

Comment:
It might be github mis-rendering this, but it looks like the indentation here is all wrong.  It should use tabs for indentation.